ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +649,109 @@ def patch_default_sc_to_non_default(self):
                 f"--request-timeout=120s"
             )
 
+    def uninstall_osc(self):
+        ocp_obj = ocp.OCP()
+        provisioners = ['openshift-storage.rbd.csi.ceph.com',
+                        'openshift-storage.cephfs.csi.ceph.com',
+                        'openshift-storage.noobaa.io/obc']
+
+        # STEP 1:List the storage classes
+        sc_list = get_all_storageclass_names(all_ns=True)
```

Comment:
you should not call a function that resides in tests/helpers as here (ocs_ci/deployment/deployment.py) is the infra for the tests.
Please move the implementation of get_all_storageclass_names to somewhere else, under ocs_ci dir

Reply:
please also change get_all_storageclass_names() so it will return OCS objects and you won't need to instantiate an OCP object here